src/Misc/layoutbin/update.cmd.template

```diff
@@ -9,6 +9,7 @@ rem  EXISTAGENTVERSION = 2.100.0
 rem  DOWNLOADAGENTVERSION = 2.101.0 
 rem  UPDATELOG = _diag/SelfUpdate-UTC.log
 rem  RESTARTINTERACTIVEAGENT = 0/1
+rem  ISINTERACTIVESESSIONCONFIGURED = 0/1
```

Comment:
when it runs as service, I don't see any sync between script laying out the files and the agent service starting the agent.listener.exe again. i'll confirm this with Ting.